.github/actions/test-khiops-install/action.yml

```diff
@@ -19,13 +21,24 @@ runs:
         OMPI_MCA_rmaps_base_oversubscribe: true
         KHIOPS_MPI_VERBOSE: true
       run: |
+        # Compute or get the khiops_env directory
+        if command -v khiops_env &>/dev/null
+        then
+          KHIOPS_ENV_DIR=$(dirname $(type -P khiops_env))
```

Comment:
Why not using `command -v` instead of `type -P` ? It should have the same output and, because of the `if` before, you are guaranteed that it will be the path of `khiops_env`

Reply:
And all below.

Reply:
```bash
$ command -v ls
alias ls='ls --color=auto'
$ type ls
ls is aliased to `ls --color=auto'
$ type -P ls
/usr/bin/ls
$ type -P foobar
$
$ echo $?
1
$ command -v foobar
$
$ echo $?
```

Hence:
- `command -v` is useful to test whether the command exists or not (gives empty output and return code 1 if the command doesn't exist), but it doesn't always return the full path to the said command
- `type -P` is useful to get the full path to an existing command (gives empty output and return code 1 if the command doesn't exist).

Hence, `type -P` seems to supersede `command -v` and could always be used IMHO, but this would require more intrusive changes to the code. I wanted to keep the code change minimal.

Reply:
OK

Reply:
Done: Using `type -P` in the whole code block which checks `khiops_env`. Everywhere needed.